### DISCUSSION TITLE
NS entries download Failed

### DISCUSSION THREAD

**Discussion by michaellack** [NightscoutFoundation/xDrip#2683] [General]
Hi new problem started this morning, I have been running companion mode for 72 days since we swapped pump to YPSOMed without any problems then started to get 502 gateway error. Checked freedns was still running and Google Cloud servers logged out of freedns and then started to get following message
![Screenshot_20230216_142040_xDrip+](https://user-images.githubusercontent.com/119345943/219391504-ae8e28e1-c6ad-4d9b-b7ad-5ba68d3e625a.jpg)

**Comment by michaellack** [comment]
Any assistance would be greatly appreciated, many thanks in advance, Michael.

**Comment by Navid200** [comment]
If you open a web browser and enter the hostname of Nightscout, can you access Nightscout?

**Comment by michaellack** [comment]
Hi, no I receive this message, 403 Forbidden
Illegal filename.  Thanks, Michael.

**Reply by Navid200** [reply]
Please log into Google Cloud and go to the status page and post a screenshot.  https://navid200.github.io/xDrip/docs/Nightscout/Status.html

Please scale the image to make it as small as possible before posting to avoid making the thread too hard to follow.

**Comment by Navid200** [comment]
Please have a look at this page:  https://navid200.github.io/xDrip/docs/Nightscout/502BadGateway.html

If you follow that and the problem is not resolved, please let me know.

**Comment by michaellack** [comment]
![Screenshot 2023-02-16 145630](https://user-images.githubusercontent.com/119345943/219399860-83c667c7-3f6b-4611-b953-5ab95982efcc.jpg)

**Reply by Navid200** [reply]
That's not the status page.
Please follow this and the problem should be resolved: https://navid200.github.io/xDrip/docs/Nightscout/502BadGateway.html

**Reply by michaellack** [reply]
Hi, thanks for all your help again, I stopped the sever and started it again and then could login to his server page and everything started working again. I will remember that in future so I don't have to bother you, thanks again, Michael.

**Reply by Navid200** [reply]
You are welcome,

There was no need to stop the server.  I am glad everything is working.
Hopefully, this will not happen again.

You can always reach out to us here for any questions about xDrip or Nightscout on Google Cloud.

**Comment by michaellack** [comment]
Thanks I will try that, Michael.
